libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp

```diff
@@ -33,13 +33,19 @@
 #include "AP_InertialSensor_Invensensev3.h"
 #include <utility>
 #include <stdio.h>
+#include <GCS_MAVLink/GCS.h>
 
 extern const AP_HAL::HAL& hal;
 
 /*
   gyro as 16.4 LSB/DPS at scale factor of +/- 2000dps (FS_SEL==0)
  */
-static const float GYRO_SCALE = (0.0174532f / 16.4f);
+static const float GYRO_SCALE_DEFAULT = (0.0174532f / 16.4f);
+/*
+  gyro as 8.2 LSB/DPS at scale factor of +/- 4000dps (FS_SEL==0)
+ */
+static const float GYRO_SCALE_ICM45686 = (0.0174532f / 8.2f);
```

Comment:
Be better to name this based on the scale rather than the name of the IMU - I am sure there will be others